entrypoint

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
Please move docker stuff into its own folder, e.g. `docker`. It'll be easier to export-ignore it in package distribution.

Reply:
This one impossible to move. Must be in the root. 
.dockerignore

Reply:
https://github.com/vimeo/psalm/pull/7701/commits/957001bfdec4bef2855fdc19dbd16fbe8920abf8